src/helpers.jl

```diff
@@ -65,3 +65,24 @@ function compare(A::AbstractArray{T}, B::AbstractArray{S}) where T<:Number where
 
   return isapprox(A,B)
 end
+
+
+# NOTE
+# vurrenlty julia/sparearrays does not implement this function (type signature)
+# once it does this can be removed/depracted in favor of mul!
+# see also: test/slice_matrices.jl
+function right_mul!(C::StridedMatrix, X::StridedMatrix, A::SparseMatrixCSC)
```

Comment:
Why not overload `LinearAlgebra.mul!`? We then only have to remove the custom definition here once it lands in Julia.